src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const { type, extraData, keysToRemove } of actions) {
```

Comment:
for better readability it can be rewritten:
```suggestion
  for (const action of actions) {
    const { type, extraData, keysToRemove }  = action;
```